cinema-model/src/main/java/touk/recru/app/repository/room/ScreeningRoomRepositoryImpl.java

```diff
@@ -0,0 +1,15 @@
+package touk.recru.app.repository.room;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+import touk.recru.app.entity.ScreeningRoom;
+
+import java.util.List;
+import java.util.Optional;
+
+@Repository
+interface ScreeningRoomRepositoryImpl extends ScreeningRoomRepository, JpaRepository<ScreeningRoom, Long> {
```

Comment:
`ScreeningRoomRepositoryImpl` to powinno raczej nazywać: `JpaScreeningRoomRepositor` ;)  Sytuacja dotyczy również innych repozytoriów.

Reply:
Zgadzam się, mój błąd z nazewnictwem.